content/src/Server.ts

```diff
@@ -44,6 +44,26 @@ export class Server {
     this.port = env.getConfig(EnvironmentConfig.SERVER_PORT)
 
     this.app = express()
+
+    const corsOptions: cors.CorsOptions = {
+      origin: true,
+      methods: 'GET,HEAD,POST,PUT,DELETE,CONNECT,OPTIONS,TRACE,PATCH',
```

Comment:
OPTIONS should not be part of the allowed methods
```suggestion
      methods: 'GET,HEAD,POST,PUT,DELETE,CONNECT,TRACE,PATCH',
```